kentikapi/client.go

```diff
@@ -132,25 +122,34 @@ func (c *Config) FillDefaults() {
 		c.APIURL = APIURLUS
 	}
 
-	if c.CloudExportHostPort == "" {
-		c.CloudExportHostPort = cloudExportHostPort
-	}
-
-	if c.SyntheticsHostPort == "" {
-		c.SyntheticsHostPort = syntheticsHostPort
-	}
-
 	if c.Timeout == nil {
 		c.Timeout = pointer.ToDuration(defaultTimeout)
 	}
 
 	c.RetryCfg.FillDefaults()
 }
 
-func makeConnForGRPC(c Config, hostPort string) (grpc.ClientConnInterface, error) {
+func parseKentikURL(apiURL *string) (string, bool) {
+	tls := false
+	u, err := url.Parse(*apiURL)
+	if err == nil {
+		*apiURL += "/api/v5"
+		grpc := u.Host
+		if u.Scheme == "https" {
+			tls = true
+			if u.Port() == "" {
+				grpc += ":443"
```

Comment:
This should not be needed, because 443 is used by default: https://github.com/grpc/grpc/blob/master/doc/naming.md

Reply:
When trying Synthetics examples without specifying port:
rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp: address grpc.api.kentik.com: missing port in address"

Reply:
Ok.
> he following schemes are supported by the gRPC C-core implementation, but may not be supported in other languages: 
> ipv4:address[:port][,address[:port],...] -- IPv4 addresses
It seems that it is not the case for grpc-go.